### DISCUSSION TITLE
Use of Prompts

### DISCUSSION THREAD

**Discussion by raviwarrier** [karakeep-app/karakeep#534] [Q&A]
version in use: 0.18.0

I wanted to customize the placeholder prompt but am a little confused. Will the app automatically replace <CONTENT_HERE> for the bookmark/url it is working on or do I need to do it manually?

also, in my opinion, it is always a good practice to give the structure as an example to ai prompts so that they return exactly what you are looking for. For example, this is from another code:

Your response should be in the following format:
    {
        \"subtopics\": [
            \"Subtopic 1\",
            \"Subtopic 2\",
            \"Subtopic 3\",
            \"Subtopic 4\",
            \"Subtopic 5\",
            \"Subtopic 6\",
            \"Subtopic 7\",
            \"Subtopic 8\",
            \"Subtopic 9\",
            \"Subtopic 10\"
        ]
    }";

this allows me to get the exact format always and don't need to worry about the output or parsing it for extra words or characters. Just a tip if you want to include it in your default placeholder prompt.

**Comment by raviwarrier** [comment]
And there's a 100 character limit on the prompt. Even the placeholder/preview prompt is longer than 100 chars. Was this an intentionally set limit?

**Reply by akshara-tg** [reply]
Even I faced the 100 character limit. https://github.com/hoarder-app/hoarder/discussions/530#discussioncomment-10942703

**Comment by kamtschatka** [comment]
You are not replacing the whole prompt, you are just adding new lines to the prompt. You can just add multiple lines there.
And no, the "<CONTENT_HERE>" does not need to be replaced by you, this will be replaced when the inference is performed.
